webapp/components/wpt-results.js

```diff
@@ -958,7 +963,11 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
         if (count >= num || !shouldRetry(err)) {
           throw err;
         }
-        window.setTimeout(retry, wait);
+        return new Promise((resolve, reject) => {
+          window.setTimeout(() => {
+            return retry().then(resolve, reject)
```

Comment:
nit: `return` is misleading. Nothing cares about the return of f'n passed to `window.setTimeout`. Should drop return, and could consider simply `() => retry().then(resolve, reject)`.